code/game/gamemodes/clock_cult/clockcult.dm

```diff
@@ -229,27 +229,27 @@ GLOBAL_VAR(clockcult_eminence)
 			if(CLOCKCULT_PREFIX_RECRUIT)
 				var/role = sender.mind?.assigned_role
 				//Ew, this could be done better with a dictionary list, but this isn't much slower
-				if(role in GLOB.command_positions)
+				if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_COMMAND))
 					prefix = "High Priest"
-				else if(role in GLOB.engineering_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_ENGINEERING))
 					prefix = "Cogturner"
-				else if(role in GLOB.medical_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_MEDICAL))
 					prefix = "Rejuvinator"
-				else if(role in GLOB.science_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_SCIENCE))
 					prefix = "Calculator"
-				else if(role in GLOB.supply_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_CARGO))
 					prefix = "Pathfinder"
 				else if(role in JOB_NAME_ASSISTANT)
 					prefix = "Helper"
 				else if(role in JOB_NAME_MIME)
 					prefix = "Cogwatcher"
 				else if(role in JOB_NAME_CLOWN)
 					prefix = "Clonker"
-				else if((role in GLOB.civilian_positions) || (role in GLOB.gimmick_positions))
+				else if((role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_CIVILIAN)))
 					prefix = "Cogworker"
-				else if(role in GLOB.security_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_SECURITY))
 					prefix = "Warrior"
-				else if(role in GLOB.nonhuman_positions)
+				else if(role in SSdepartment.get_jobs_by_dept_id(DEPARTMENT_SILICON))
```

Comment:
These titles should be in job datum when you're going to make `var/datum/job/assigned_role` in this refactor,